lib/external-housing-available/external-housing-available.controllers.js

```diff
@@ -1,6 +1,172 @@
+        addressObject.addressCounty !== process.env.ADDRESS_LIMIT_COUNTY
+      ) {
+        try {
+          await Log.request({
+            type: 'externalhousingaddress.submit.failed.county',
+            severity: 'Notice',
+            request: request,
+          }); 
+        } catch (e) {} 
+        throw Boom.badData('Sorry, We are only accepting properties in ' + `${process.env.ADDRESS_LIMIT_COUNTY}, ${process.env.ADDRESS_LIMIT_STATE}`); 
+      }
+
+      //Extracting the address using function from geoCoordinates lib
+      let geoCoordObject = geoCoordinatesLib.extractLocation(firstGeocode);
+      
+      const addressHash = postalAddressLib.hashAddress(addressObject);
+      addressObject.hash = addressHash;
+
+      // External Housing Available Object 
+      let externalHousingAvailableObject = {};
+
+      const existingAddresses = await postalAddressLib.getAddressesByHash(addressHash);
+      if (existingAddresses.length > 0) {
+        //If this is an exisiting address
+        externalHousingAvailableObject.PropertyId = existingAddresses[0].PropertyId;
+      } else {
+        //Else, create a new propertyId for non exisiting address
+      }
+
+      const standardizedAddress = postalAddressLib.standardizeAddress(addressObject);
+      addressObject.address = standardizedAddress; 
+
+      //Title
+      if (request.payload.title && request.payload.title !== request.payload.address) {
+        externalHousingAvailableObject.title = request.payload.title;
+      } else {
+        externalHousingAvailableObject.title = standardizedAddress; 
+      }
+      //AuthorId
+      if (request.payload.AuthorId && request.auth.credentials.subjectId === process.env.SUPER_ADMIN) {
+        externalHousingAvailableObject.AuthorId = request.payload.AuthorId;
+      } else {
+        throw Boom.badRequest('ExternalHousingAvailable.AuthorId must match an Super Admin account ID.');
+      }
+      //Address
+      if (standardizedAddress) {
+        externalHousingAvailableObject.address = standardizedAddress;
+      } 
+      //Beds
+      if (request.payload.beds) {
+        let beds = Number(request.payload.beds); 
+        externalHousingAvailableObject.beds = beds; 
+      } 
+      //Baths
+      if (request.payload.baths) {
+        let baths = Number(request.payload.baths);
+        externalHousingAvailableObject.baths = baths;
+      } 
+      //Contacts 
+      if (request.payload.contact) {
+        externalHousingAvailableObject.contact = request.payload.contact;
+      }
+      //Body
+      if (request.payload.body) {
+        externalHousingAvailableObject.body = request.payload.body;
+      }
+      //Url
+      if (request.payload.url) {
+        externalHousingAvailableObject.url = request.payload.url;
+      }
+      //Status
+      if (request.payload.status) {
+        externalHousingAvailableObject.status = request.payload.status;
+      } else {
+        externalHousingAvailableObject.status = "N/A";
```

Comment:
let's default the status to 'active' if it isn't provided by the admin. we may want to add a state of 'unknown' later, but I think 'active' will do for now.